From 1383f5df74eda98b937bcf7001c80cfcd7d6b732 Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Thu, 22 Feb 2024 14:10:43 +0100
Subject: [PATCH] dt-bindings: display: update dt-bindings for STM32 LTDC
 device

Add "st,stm32mp25-ltdc" compatible for SOC MP25

Change-Id: Iacf9bc09ccc610bb9b048c39ab225db3b7210e4a
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
---
 Documentation/devicetree/bindings/display/st,stm32-ltdc.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/display/st,stm32-ltdc.yaml b/Documentation/devicetree/bindings/display/st,stm32-ltdc.yaml
index d6ea4d62a2cf..b8c1ad083a9d 100644
--- a/Documentation/devicetree/bindings/display/st,stm32-ltdc.yaml
+++ b/Documentation/devicetree/bindings/display/st,stm32-ltdc.yaml
@@ -13,6 +13,7 @@ maintainers:
 properties:
   compatible:
     const: st,stm32-ltdc
+    const: st,stm32mp25-ltdc
 
   reg:
     maxItems: 1
-- 
2.39.5

